README.md

```diff
@@ -1,4 +1,110 @@
-# large-events
+# Large Events
 
 [![Build Status](https://travis-ci.com/knative-portability/large-events.svg?branch=master)](https://travis-ci.com/knative-portability/large-events)
 [![Coverage Status](https://coveralls.io/repos/github/knative-portability/large-events/badge.svg?branch=master)](https://coveralls.io/github/knative-portability/large-events?branch=master)
+
+## Introduction
+
+An app built as proof of portability for [Knative](https://knative.dev) that provides a platform for event organizers to share info during large events like concerts, parades, expos, etc. Key features of this app include:
+* Allow authorized event organizers to create and edit events
+* Handle uploading and serving multimedia posts made by attendees
+* Organize and filter multiple sub-events
+* Provide a clean front-end web UI for interaction with the service
+* Authenticate with OAuth 2.0
+
+This app is built as part of a proof of portability project for [Knative](https://knative.dev). It is meant to show the key features of Knative, to test the conformance various cloud product implementations of Knative, and to document with functioning sample code how one might develop, build, and deploy with Knative. Key features of Knative this app demonstrates include:
```

Comment:
> "to test the conformance various cloud product implementations of Knative"
slight typo but could be confusing to readers - should it be 'conformance across various ..." or something similar?

Reply:
Thank you, fixed in https://github.com/knative-portability/large-events/pull/7